quickwit-indexing/src/actors/packager.rs

```diff
@@ -102,15 +102,15 @@ impl Actor for Packager {
     }
 
     fn queue_capacity(&self) -> QueueCapacity {
-        QueueCapacity::Bounded(0)
+        QueueCapacity::Bounded(1)
```

Comment:
This had an impact on indexing throughput.

Reply:
On the contrary, we will consume more RAM.

Reply:
Yes I saw a real difference in throughput. I don't quite understand why to be honest. 
Probably some race conditions due to the limitation of the thread pool.

(Indexer just needs to pass this value to the packager to yield while Packager is waiting for someone to yield.)